docs/docs/Components/components-processing.md

```diff
@@ -171,6 +171,93 @@ This component can perform the following operations on Pandas [DataFrame](https:
 
 </details>
 
+## Data operations
+
+This component performs operations on [Data](/concepts-objects#data-object) objects, including selecting keys, evaluating literals, combining data, filtering values, appending/updating data, removing keys, and renaming keys.
+
+1. To use this component in a flow, connect a component that outputs [Data](/concepts-objects#data-object) to the **Data Operations** component's input.
+2. In the **Operations** field, select the operation you want to perform.
+For example, send this request to the **Webhook** component.
+Replace `YOUR_FLOW_ID` with your flow ID.
+```bash
+curl -X POST "http://127.0.0.1:7860/api/v1/webhook/YOUR_FLOW_ID" \
+-H 'Content-Type: application/json' \
+-d '{
+  "id": 1,
+  "name": "Leanne Graham",
+  "username": "Bret",
+  "email": "Sincere@april.biz",
+  "address": {
+    "street": "Kulas Light",
+    "suite": "Apt. 556",
+    "city": "Gwenborough",
+    "zipcode": "92998-3874",
+    "geo": {
+      "lat": "-37.3159",
+      "lng": "81.1496"
+    }
+  },
+  "phone": "1-770-736-8031 x56442",
+  "website": "hildegard.org",
+  "company": {
+    "name": "Romaguera-Crona",
+    "catchPhrase": "Multi-layered client-server neural-net",
+    "bs": "harness real-time e-markets"
+  }
+}'
+```
+
+3. In the **Data Operations** component, select the **Select Keys** operation to extract specific user information.
+To add additional keys, click <Icon name="Plus" aria-label="Add"/> **Add More**.
+![A webhook and data operations component](/img/component-data-operations-select-key.png)
+4. Filter by `name`, `username`, and `email` to select the values from the request.
+```json
+{
+  "name": "Leanne Graham",
+  "username": "Bret",
+  "email": "Sincere@april.biz"
+}
+```
+
+### Operations
+
+The component supports the following operations:
+
+| Operation | Required Inputs | Description |
+|-----------|----------------|-------------|
+| Select Keys | `select_keys_input` | Selects specific keys from the data. |
+| Literal Eval | None | Evaluates string values as Python literals. |
+| Combine | None | Combines multiple data objects into one. |
+| Filter Values | `filter_key`, `filter_values`, `operator` | Filters data based on key-value pair. |
+| Append or Update | `append_update_data` | Adds or updates key-value pairs. |
+| Remove Keys | `remove_keys_input` | Removes specified keys from the data. |
+| Rename Keys | `rename_keys_input` | Renames keys in the data. |
+
+<details>
+<summary>Parameters</summary>
+
+**Inputs**
+
+| Name | Display Name | Info |
```

Comment:
Why do you use "Info" as the column name for inputs and outputs, but "Description" for the operations?

Reply:
Changed to info